src/main/java/com/researchspace/api/v1/InventoryFilesApi.java

```diff
@@ -65,4 +67,9 @@ ApiInventoryFile uploadFile(MultipartFile file, ApiInventoryFilePost settings, U
 
   @DeleteMapping(value = "/{id}")
   ApiInventoryFile deleteFile(Long id, User user);
+
+  @GetMapping("/image/{contentsHash}")
+  ResponseEntity<byte[]> getImageByContentsHash(
+      @PathVariable String contentsHash, @RequestAttribute(name = "user") User user)
```

Comment:
It turns out actually since spring 5 annotations on the method params on Controller interfaces are honoured https://www.baeldung.com/spring-interface-driven-controllers. 

I've removed `@RequestAttribute` to match the rest of the methods. If I remove `@PathVariable` though it generates a warning in IntelliJ. I think it makes sense to define the `@PathVariable` at the same place where the `@GetMapping` with the placeholder value is though i.e. on the interface.